### PR TITLE
CTOR-1143 : Refresh network-stormshield-ssh

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh network-stormshield-ssh

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] 25.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-stormshield-ssh.md [hunk 1]
```diff
@@ -2,34 +2,344 @@
...
+
+Le connecteur apporte le modèle de service suivant
+(classé selon le modèle d'hôte auquel il est rattaché) :
+
+<Tabs groupId="sync">
+<TabItem value="Net-Stormshield-SSH-custom" label="Net-Stormshield-SSH-custom">
+
+| Alias     | Modèle de service                    | Description                |
+|:----------|:-------------------------------------|:---------------------------|
+| Qos-Usage | Net-Stormshield-Qos-Usage-SSH-custom | Controle l'utilisation QoS |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-stormshield-ssh.md:30]:
```suggestion
| Qos-Usage | Net-Stormshield-Qos-Usage-SSH-custom | Contrôle l'utilisation QoS |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-stormshield-ssh.md [hunk 2]
```diff
@@ -2,34 +2,344 @@
...
+telle que celle-ci (remplacez les valeurs d'exemple par les vôtres) :
+
+```bash
+/usr/lib/centreon/plugins/centreon_stormshield_ssh.pl \
+	--plugin=network::stormshield::local::plugin \
+	--mode=qos-usage \
+	--hostname='10.0.0.1' \
+	--ssh-backend='libssh' \
+	--ssh-username='' \
+	--ssh-password='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-stormshield-ssh.md:222]: ssh username and password

File: pp/integrations/plugin-packs/procedures/network-stormshield-ssh.md [hunk 3]
```diff
@@ -2,34 +2,339 @@
...
+is able to monitor a resource using a command like this one (replace the sample values by yours):
+
+```bash
+/usr/lib/centreon/plugins/centreon_stormshield_ssh.pl \
+	--plugin=network::stormshield::local::plugin \
+	--mode=qos-usage \
+	--hostname='10.0.0.1' \
+	--ssh-backend='libssh' \
+	--ssh-username='' \
+	--ssh-password='' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-stormshield-ssh.md:217]: ssh username and password

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-stormshield-ssh.md (hunk 2 above)
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-stormshield-ssh.md:222]:
```suggestion
	--ssh-username='XXXX' \
	--ssh-password='XXXX' \
```

File: pp/integrations/plugin-packs/procedures/network-stormshield-ssh.md (hunk 3 above)
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-stormshield-ssh.md:217]:
```suggestion
	--ssh-username='XXXX' \
	--ssh-password='XXXX' \
```

**Pull request merged**